content/docs/glossary/claimable-balance.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Claimable Balance
+order:
+---
+
+import { CodeExample } from "components/CodeExample";
+
+Claimable Balances can be used to "split up" a payment into two parts, which allows the sending to only depend on the sending account, and the receipt to only depend on the receiving account. An account can initiate the "send" by creating a ClaimableBalanceEntry with [Create Claimable Balance](../start/list-of-operations.mdx#create-claimable-balance), and then that entry can be claimed by the claimants specified on the ClaimableBalanceEntry at a later time with [Claim Claimable Balance](../start/list-of-operations.mdx#claim-claimable-balance). 
+
+## Relevant operations
+
+[Create Claimable Balance](../start/list-of-operations.mdx#create-claimable-balance)
+
+### Parameters
+
+1. Asset - Asset that will be held in the ClaimableBalanceEntry.
+
+2. Amount - Amount of asset stored in the ClaimableBalanceEntry.
+
+3. List of Claimants - A Claimant is an object that holds both the account that can claim the ClaimableBalanceEntry, and a ClaimPredicate that must evaluate to true for the claim to succeed. A ClaimPredicate is a recursive data structure that can be used to construct complex conditionals. Here are some examples -
```

Comment:
Some kind of formatting on the "vocab term" itself (e.g. **Asset**) would make it pop out and also make it clearer that you're referencing it below when you say "will move Amount of Asset"

Reply:
Done